src/garage/sampler/is_sampler.py

```diff
@@ -180,3 +235,9 @@ def _sample_isweighted_paths(self,
                 return []
 
         return samples
+
+    def start_worker(self):
```

Comment:
these methods could go in `BaseSampler`, allowing you to not define these stubs everywhere.